app/src/main/java/fi/thl/koronahaavi/exposure/ExposureUpdateWorker.kt

```diff
@@ -50,7 +48,6 @@ class ExposureUpdateWorker @WorkerInject constructor(
 
                 // this call will trigger EN system notifications (only on real use)
                 exposureNotificationService.getExposureDetails(token)
-                    .map(ExposureInformation::toExposure)
```

Comment:
this mapping to Exposure domain object is now done in ExposureNotificationService implementation